### PR TITLE
Adding translations to appdata

### PR DESCRIPTION
**Pull request by LeandroStanger**

### CODE REVIEW

**Comment by nyalldawson** [comment]
The spell checker choked on this -- I've made it skip this file now.

**Comment by nyalldawson** [comment]
@aleixpol look correct to you?

File: linux/org.qgis.qgis.appdata.xml [hunk 1]
```diff
@@ -2,11 +2,17 @@
...
+  </screenshots>
+  <description>QGIS is a user friendly Open Source Geographic Information System (GIS) licensed under the GNU General Public License. QGIS is an official project of the Open Source Geospatial Foundation (OSGeo). It runs on Linux, Unix, Mac OSX, Windows and Android and supports numerous vector, raster, and database formats and functionalities.</description>
+  <description xml:lang="pt_BR">O QGIS é um Sistema de Informação Geográfica (SIG) de Código Aberto licenciado segundo a Licença Pública Geral GNU. O QGIS é um projeto oficial da Open Source Geospatial Foundation (OSGeo). Funciona em Linux, Unix, Mac OSX, Windows e Android e suporta inúmeros formatos de vetores, rasters e bases de dados e funcionalidades.</description>
+  <description xml:lang="es">QGIS es un Sistema de Información Geográfica (SIG) de Código Abierto licenciado bajo GNU - General Public License . QGIS es un proyecto oficial de Open Source Geospatial Foundation (OSGeo). Corre sobre Linux, Unix, Mac OSX, Windows y Android y soporta numerosos formatos y funcionalidades de datos vector, datos ráster y bases de datos.</description>
```
Comment (aleixpol) [inline linux/org.qgis.qgis.appdata.xml]: There's a dangling space between License and the dot.

**Review by aleixpol — COMMENTED** [review]
Looks good to me. In KDE we have this automatised so GUI translators can also translate these fields, maybe you'll want to do the same.
Or updating the appdata info by hand is always an option I guess.

**Pull request merged**